x-pack/legacy/plugins/monitoring/public/np_imports/angular/modules.ts

```diff
@@ -5,6 +5,8 @@
  */
 
 import angular, { IWindowService } from 'angular';
+// required for `ngSanitize` angular module
+import 'angular-sanitize';
```

Comment:
I actually already have this in my NP pr [here](https://github.com/elastic/kibana/pull/62908/files#diff-39137a7f568d2da312afefa2144ff12bR9) Maybe this can be removed then? (to avoid merge conflicts). I can just add the comment so it follows the same pattern. wdyt?

Reply:
I don't think this will cause merge conflicts if the change is the same, and we probably shouldn't add a dependency on that PR to make this PR "complete"

Reply:
It's the same, but I moved everything outside the `legacy` directory (and the file name is also different) thus github won't track it as same (and won't do its "auto-merge" magic).

Just thought I should bring it up, so you're not confused with the deleted `modules.ts`  file when trying to merge and resolve conflicts (if my PR goes into master first that is)